### PR TITLE
⚠️ DON'T MERGE BEFORE MAINNET - Adjusting READMEs for mainnet release

### LINKED ISSUE #1785: Update Readme for the mainnet release
## Description
Update Readme for the mainnet release

## Acceptance criteria
- The readme says that it has been released to mainnet (official wording)
- The bug bounty is mentioned (link to the bug bounty website)
- Steps to make a new transfer is updated (also referencing the new documentation)
- The needs to be a separate pull request which is getting merged for the mainnet release

## Tasks
- [x] Check if the SDK documentation deployment works as expected when clicking links @agatsoh 
- [x] Update connection documentation for the dApp with the new alternatives provided by the connection manager

### PR DESCRIPTION
**Pull request by taleldayekh**
**Thank you for submitting this pull request :)**

Fixes #1785 

**Short description**

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [ ] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,87 +136,112 @@ Look at the [Raiden Light Client SDK folder of this repository](./raiden-ts/READ
...
 
-#### 1. Acquire ETH
+### Making a First Transfer
 
-1. Go to the Goerli faucet at [htts://faucet.goerli.mudit.blog](https://faucet.goerli.mudit.blog) or [https://goerli-faucet.slock.it/](https://goerli-faucet.slock.it/)
-2. Follow the instructions on how to acquire Goerli ETH on respective website
+1. **Connect to the dApp**  
 
-#### 2. Navigate to the Raiden Hub
+   - Mainnet
```
Comment (christianbrb) [inline README.md:170]: A good approach to have the two guides here :) I would make two separate sections as an outer braket. First for mainnet and second for testnet.

File: README.md [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
+      alt='Raiden Logo' 
       src="https://user-images.githubusercontent.com/35398162/54018436-ee3f6300-4188-11e9-9b4e-0666c44cda53.png" /></a>
   <br/>
   Raiden Light Client SDK, CLI and dApp
   <br/>
 </h2>
 
 <h4 align="center">
-  JavaScript SDK, CLI and dApp to carry out fast, cheap, scalable off-chain token transfers with other <a href="https://github.com/raiden-network/raiden">Raiden Clients</a>.
+  JavaScript SDK, CLI and dApp to carry out fast, cheap, scalable off-chain token transfers with other <a href="https://github.com/raiden-network/raiden">Raiden Clients on Ethereum testnets and mainnet.</a>.
```
Comment (christianbrb) [inline README.md:15]:
```suggestion
  JavaScript SDK, CLI and dApp to carry out fast, cheap, scalable off-chain token transfers with other <a href="https://github.com/raiden-network/raiden">Raiden Clients</a>.
  
 I would delete it here
```

File: README.md [hunk 3]
```diff
@@ -44,31 +45,31 @@
 
 The Raiden Light Client SDK is a [Raiden Network](https://raiden.network) compatible client written in JavaScript/Typescript, capable of running in modern web3-enabled browsers, wallets and Node.js environments.
 
-The Raiden CLI is a reference implementation which provides a HTTP REST server that is fully compatible with the [Raiden API specification](https://docs.raiden.network/raiden-api-1/resources).
+The Raiden CLI is a reference implementation that provides a HTTP REST server which is fully compatible with the [Raiden API specification](https://docs.raiden.network/raiden-api-1/resources).
 
 The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Light Client SDK, which can be used with web3 wallets like [Metamask](https://metamask.io/) (Desktop) or [imToken](https://token.im/download) (mobile).
 
-> **INFO:** The Light Client SDK, CLI and dApp are all **work in progress** projects.
+> **INFO:** The Light Client SDK, CLI and dApp are all **work in progress** projects. All three projects have been released for mainnet and all code is available in the [Light Client repository](https://github.com/raiden-network/light-client).
```
Comment (christianbrb) [inline README.md:52]:
```suggestion
> **INFO:** The Light Client SDK, CLI and dApp are all **work in progress** projects. All three projects have been released for mainnet and all code is available in the [Light Client repository](https://github.com/raiden-network/light-client).  As this release still has its limitations and is a beta release, it is crucial to read this readme including the security notes carefully before using the software.
```

**Review by christianbrb — CHANGES_REQUESTED** [review]

**Comment by christianbrb** [comment]
@taleldayekh Nice, looks already good. I have requested a few changes.

Please also add changes to 
- The dApp readme: https://github.com/raiden-network/light-client/blob/master/raiden-dapp/README.md
- The SDK readme: https://github.com/raiden-network/light-client/blob/master/raiden-ts/README.md
- The CLI / API readme: https://github.com/raiden-network/light-client/blob/master/raiden-cli/README.md

... but maybe wait for #1947 to be implemented to avoid merge conflicts.

We need to think about adding
1. Requirements for the safe usage
2. The changed disclaimer 

There are internal issues for this.

File: README.md (hunk 1 above)
Comment (taleldayekh) [reply README.md:172]: Ok, I'll split it into two guides.

**Comment by weilbith** [comment]
The linting issue got fixed. There can't be an issue in the CI now. Therefore skipping early and merge it for the final release.

**Review by weilbith — APPROVED** [review]

**Comment by weilbith** [comment]
I fixed the linting issue. It is not possible that CI fails now. So because of pressuring mainnet release we merge it early.

**Pull request merged**